### PR TITLE
feat(core): adjust ai help island style and behavior, add animation

### PR DESCRIPTION
**Pull request by CatsJuice**
- Move right-sidebar activeTab state into `RightSidebarService`
- Remove `styled` usage and adjust the UI
- Fix the issue that ai-button clicking not work when sidebar opened
- Add an animation if AI-Chat panel hasn't been opened.

### CODE REVIEW

**Comment by CatsJuice** [comment]
* **#7310** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7310?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @CatsJuice and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by JimmFly — APPROVED** [review]

File: packages/frontend/core/src/components/pure/ai-island/index.tsx [hunk 1]
```diff
@@ -1,65 +1,66 @@
...
+  borderAngle1,
+  borderAngle2,
+  borderAngle3,
+} from './styles.css';
 
-export const RIGHT_SIDEBAR_TABS_ACTIVE_KEY =
-  'app:settings:rightsidebar:tabs:active';
+if (typeof window !== 'undefined' && window.CSS) {
+  const getName = (nameWithVar: string) => nameWithVar.slice(4, -1);
+  const registerAngle = (varName: string, initialValue: number) => {
```
Comment (pengx17) [inline packages/frontend/core/src/components/pure/ai-island/index.tsx:18]: what is the use case for defining css var using registerProperty here?

Comment (CatsJuice) [reply packages/frontend/core/src/components/pure/ai-island/index.tsx:18]: animate gradient angle

Comment (pengx17) [reply packages/frontend/core/src/components/pure/ai-island/index.tsx:18]: I mean this is the first time we use the `registerProperty` api. Will animation still work without using it?

Comment (CatsJuice) [reply packages/frontend/core/src/components/pure/ai-island/index.tsx:18]: Yes it's the first time. Transition for gradient color won't work without this. But actually we can use `transform: rotate()` alternatively.

**Review by pengx17 — APPROVED** [review]

**Pull request merged**